executor/explain_unit_test.go

```diff
@@ -71,22 +71,26 @@ func TestExplainAnalyzeInvokeNextAndClose(t *testing.T) {
 		explain:      nil,
 	}
 	// mockErrorOperator returns errors
-	mockOper := mockErrorOperator{baseExec, false, false}
-	explainExec.analyzeExec = &mockOper
+	mockOpr := mockErrorOperator{baseExec, false, false}
```

Comment:
Why rename it?

Reply:
Because the abbreviation of the operator is Opr instead of Oper.